README.md

```diff
@@ -286,14 +259,15 @@ proceedings to the account specified by `--miner.etherbase`. You can further tun
 by changing the default gas limit blocks converge to (`--miner.targetgaslimit`) and the price
 transactions are accepted at (`--miner.gasprice`).
 
+-->
 ## Contribution
 
 Thank you for considering to help out with the source code! We welcome contributions
 from anyone on the internet, and are grateful for even the smallest of fixes!
 
-If you'd like to contribute to go-ethereum, please fork, fix, commit and send a pull request
+If you'd like to contribute to go-quai, please fork, fix, commit and send a pull request
 for the maintainers to review and merge into the main code base. If you wish to submit
-more complex changes though, please check up with the core devs first on [our Discord Server](https://discord.gg/invite/nthXNEv)
+more complex changes though, please check up with the core devs first on [our Discord Server](https://discord.gg/Nd8JhaENvU)
```

Comment:
this discord link should be the invite from the webpage

Reply:
https://discord.gg/ngw88VXXnV